tokyo-night.tmux

```diff
@@ -57,5 +58,5 @@ tmux set -g window-status-current-format "#[fg=#44dfaf,bg=#1F2335]   #[fg=#a9
 tmux set -g window-status-format "#[fg=#c0caf5,bg=default,none,dim]   $window_number #W#[nobold,dim]#{?window_zoomed_flag, $zoom_number, $custom_pane}#[fg=yellow,blink] #{?window_last_flag,󰁯 ,} "
 
 #+--- Bars RIGHT ---+
-tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B]  %Y-%m-%d #[]❬ %H:%M $git_status$wb_git_status"
+tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B] $netspeed $git_status$wb_git_status"
```

Comment:
Restore date/time widget and remove spaces of netspeed. Also, I'm planning to move date/time to the far left. So the line should be this.


```
tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B]$netspeed$git_status$wb_git_status#[fg=#a9b1d6,bg=#24283B]  %Y-%m-%d #[]❬ %H:%M "
```

I recommend putting netspeed after fixed length widgets because it could be anoying if the text continually jump around